css/style.css

```diff
@@ -1,3 +1,45 @@
+.main {
+  font-family: 'Dosis', sans-serif;
+}
+header {
+  text-align: center;
+  color: black;
+}
+header a {
+  color: black;
+  text-decoration: none;
+  display: inline;
+  padding: 10px;
+  border: 1px solid darkgray;
+
+}
 h1 {
-  color: salmon;
+  color: darkgray;
+  text-align: center;
+}
+h2 {
+  font-size: 15px;
+  color: darkgray;
+  text-align: center;
+  padding: 1px;
+}
+p {
+    position: absolute;
+    color: white;
+    font-weight: bold;
+    z-index: inherit;
+}
+.item img {
+  position: static;
+}
+
+.item {
+    position: relative;
+    padding: 10px;
+    zoom: 30%;
+}
+.parent1, .parent2 {
```

Comment:
I think you could accomplish the same result with just using one `parent` class.